commit b47071faf263a29bf9d6bf33d53171cc0cf4d70d (refs/changes/44/2944/1)
Author: Jared Morrow <jm@joyent.com>
Date:   2017-11-14T15:31:02-08:00 (1 year, 11 months ago)
    
    joyent/node-zkstream#26 zkstream will not connect to remote ZK servers

diff --git a/README.md b/README.md
index 76d326e..5363726 100644
--- a/README.md
+++ b/README.md
@@ -14,8 +14,9 @@ Opens a new ZooKeeper connection and returns the Client instance.
 
 Parameters:
  - `options`: an Object, with keys:
-   - `host`: a String, hostname or IP to connect to
-   - `port`: an optional Number
+   - `address`: a String, hostname or IP to connect to
+   - `port`: an optional Number (default: 2181)
+   - `sessionTimeout`: an optional Number (default: 30000 ms)
 
 ### `Client#close([cb])`
 
@@ -228,4 +229,3 @@ Objects, with keys:
    - `id`: a String
  - `perms`: an Array of Strings, can be `'read'`, `'write'`, `'create'`,
             `'delete'`, `'admin'`
-
diff --git a/lib/connection-fsm.js b/lib/connection-fsm.js
index cfdcdef..c4c1488 100644
--- a/lib/connection-fsm.js
+++ b/lib/connection-fsm.js
@@ -90,7 +90,7 @@ ZKConnectionFSM.prototype.state_connecting = function (S) {
 	this.zcf_log.trace('attempting new connection');
 
 	this.zcf_socket = mod_net.connect({
-		host: this.zcf_server.host,
+		host: this.zcf_server.address,
 		port: this.zcf_server.port,
 		allowHalfOpen: true
 	});
diff --git a/package.json b/package.json
index a9d8f23..9e033e9 100644
--- a/package.json
+++ b/package.json
@@ -2,6 +2,10 @@
   "name": "zkstream",
   "version": "0.9.0",
   "description": "",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/joyent/node-zkstream.git"
+  },
   "main": "lib/index.js",
   "dependencies": {
     "assert-plus": "^1.0.0",
